docs/medical-api/more-info/webhooks.mdx

```diff
@@ -359,3 +359,47 @@ Example payload:
   ]
 }
 ```
+
+#### Bulk Document Download
+
+While you can download specified documents using the [get document url endpoint](/medical-api/api-reference/document/bulk-document-download), 
+patients can have a large number of documents, and it can be tedious to download one at a time. 
+
+Instead trigger a [bulk get document url](/medical-api/api-reference/document/bulk-document-download) query - one query per patient at any given time;
+When the result is ready, we’ll send a webhook message with all the downloadable urls for all the patients documents.
```

Comment:
- let's be consistent with our use of caps
- rather than calling an endpoint tedious, perhaps we can put a positive spin on things:

If you want to download all of a patient's documents, especially when they have a large volume of documents, you can start a [Bulk Get Document URL](/medical-api/api-reference/document/bulk-document-download) query - we’ll send a webhook message with all the downloadable URLs for all the patients documents.